client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -131,16 +139,28 @@ export class DisplayTable extends React.Component {
             (value) => <span> {value} </span>, <- (function)  If it's function, column value is passed in
           raw_formatter: (value) => Dept.lookup(value).name <- (function) Actual raw value from data. Used for searching/csv string Default to value
           sum_func: (sum, value) => ... <- (function) Custom sum func. Default to sum + value
-          sort_func: (a, b, this.state.descending) => ... <- (function) Custom sort func. Default to _.sortBy
+          sort_func: (a, b) => ... <- (function) Custom sort func. Default to _.sortBy
           sum_initial_value: 0 <- (number) Default to 0
           visibility_toggleable: true <- (boolean) Defaults to false for index 0, true for all other indexes.
         },
       }
       */,
+      page_size_increment,
       util_components,
+      show_pagination_controls,
+      disable_pagination,
```

Comment:
Hmm I think `show_pagination_controls` and `disable_pagination` is redundant. I think we can just remove `disable_pagination`, and rename `show_pagination_controls` to something like `enable_pagination`?

Because it doesn't really make any difference to show pagination ctrls with disabled pagination, and also vica versa.